docs/content/kubernetes-primer.md

```diff
@@ -495,16 +495,18 @@ main = {
   "response": response,
 }
 
-default response = {"allowed": true}
-
 response = {
+    "uid": input.request.uid,
     "allowed": false,
     "status": {
         "message": reason,
     },
 } {
     reason = concat(", ", admission.deny)
     reason != ""
+} else = {
+    "uid": input.request.uid,
+    "allowed": true,
 }
```

Comment:
There are many ways to do that -- another one would be

```rego
response["uid"] = input.request.uid
response["allowed"] = allowed
response["status"] = status

default allowed = true
allowed = false {
	count(deny) > 0
}

status = { "message": reason } {
	not allowed
	reason := concat(", ", deny)
}
```

`response` is a partial object rule, where `uid` and `allowed` would always be set.